src/NuGet.Clients/NuGet.PackageManagement.UI/Options/GeneralOptionControl.Designer.cs

```diff
@@ -36,14 +36,12 @@ private void InitializeComponent()
             this.BindingRedirectsHeader = new System.Windows.Forms.Label();
             this.localsCommandButton = new System.Windows.Forms.Button();
             this.localsCommandStatusText = new System.Windows.Forms.RichTextBox();
-#if !VS14
```

Comment:
The designer automatically removed those, will go back and make sure unnecessary changes are not done like that